### PR TITLE
MAUI-1860 : Swiping feature UG Documentation

### PR DESCRIPTION
**Pull request by RiyashameedM**
Prepared UG documentation for Swiping feature

### CODE REVIEW

**Comment by RiyashameedM** [comment]
rebuild

**Comment by RiyashameedM** [comment]
rebuild

**Comment by RiyashameedM** [comment]
rebuild

File: MAUI/ListView/swiping.md [hunk 1]
```diff
@@ -0,0 +1,561 @@
...
+
+  grid.Children.Add(grid1);
+  grid.Children.Add(grid2, 1, 0);
+                
+  return grid;
+});
+{%endhighlight %}
+{% endtabs %}
+
+![Customize Start swipe in .NET MAUI Listview](Images/swiping/maui-listview-working-with-multiple-views-in-swipetemplate.jpg)
```
Comment (jayaleshwari) [inline MAUI/ListView/swiping.md:163]: Customize start swipe view in .NET MAUI ListView

Comment (jayaleshwari) [reply MAUI/ListView/swiping.md:163]: .NET MAUI ListView - use this for all the places wherever used.

File: MAUI/ListView/swiping.md [hunk 2]
```diff
@@ -0,0 +1,561 @@
...
+<td>item data</td>
+<td>Represents the item data of list view item.</td>
+</tr>
+<tr>
+<td>offset</td>
+<td>Represents the value that how much the item need to swipe.</td>
+</tr>
+</table>
+
+The `SwipeOffset` value should be positive for the left swiping of the listview item.
```
Comment (jayaleshwari) [inline MAUI/ListView/swiping.md:311]: change left to start

File: MAUI/ListView/swiping.md [hunk 3]
```diff
@@ -0,0 +1,561 @@
...
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void LeftSwipeButton_Clicked(object sender, EventArgs e)
+{
+    ListView.SwipeItem(viewModel.contactsinfo[1], 200);
+}
+{% endhighlight%}
+{% endtabs %}
+
+The `SwipeOffset` value should be negative for the right-swiping of the listview item.
```
Comment (jayaleshwari) [inline MAUI/ListView/swiping.md:322]: change right to end

File: MAUI/ListView/swiping.md [hunk 4]
```diff
@@ -0,0 +1,561 @@
...
+{% tabs %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void RightSwipeButton_Clicked(object sender, EventArgs e)
+{
+    ListView.SwipeItem(viewModel.contactsinfo[1], -200);
+}
+{% endhighlight %}
+{% endtabs %}
+
+![Programmatic Start and End swiping in .NET MAUI Listview](Images/swiping/maui-listview-programmatic-swiping.gif)
```
Comment (jayaleshwari) [inline MAUI/ListView/swiping.md:333]: start and end can lower case.

Comment (jayaleshwari) [reply MAUI/ListView/swiping.md:333]: ListView

**Review by jayaleshwari — CHANGES_REQUESTED** [review]
Please address review comments

File: MAUI/ListView/swiping.md [hunk 5]
```diff
@@ -0,0 +1,561 @@
...
+
+    grid.Children.Add(grid1);
+
+    return grid;
+
+});
+{% endhighlight %}
+{% endtabs %}
+
+N> Similarly, the UI for swiping from right to left or bottom to top in vertical or horizontal Orientation can be customized by using the `SfListView.EndSwipeTemplate`.
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/ListView/swiping.md:69]: Modified

File: MAUI/ListView/swiping.md (hunk 1 above)
Comment (RiyashameedM) [reply MAUI/ListView/swiping.md:163]: modified

File: MAUI/ListView/swiping.md (hunk 4 above)
Comment (RiyashameedM) [reply MAUI/ListView/swiping.md:333]: Modified

File: MAUI/ListView/swiping.md [hunk 6]
```diff
@@ -0,0 +1,561 @@
...
+
+N> The Swipe Template is mandatory to perform swiping in the `SfListView`.
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/ListView/swiping.md:71]: Modified

File: MAUI/ListView/swiping.md [hunk 7]
```diff
@@ -0,0 +1,561 @@
...
+
+![Start swiping functionalities](Images/swiping/maui-listview-startswipetemplate.jpg)
+
+Download the entire source code from GitHub [here](https://github.com/SyncfusionExamples/how-to-customize-the-swipe-view-in-.net-maui-listview). 
+
+![End swiping functionalities](Images/swiping/maui-listview-endswipetemplate.jpg)
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/ListView/swiping.md:77]: Modified

File: MAUI/ListView/swiping.md [hunk 8]
```diff
@@ -0,0 +1,561 @@
...
+
+});
+{% endhighlight %}
+{% endtabs %}
+
+N> Similarly, the UI for swiping from right to left or bottom to top in vertical or horizontal Orientation can be customized by using the `SfListView.EndSwipeTemplate`.
+
+N> The Swipe Template is mandatory to perform swiping in the `SfListView`.
+
+![Start swiping functionalities](Images/swiping/maui-listview-startswipetemplate.jpg)
```
[the remark this answers is not in the document]
Comment (RiyashameedM) [reply MAUI/ListView/swiping.md:73]: Modified

File: MAUI/ListView/swiping.md [hunk 9]
```diff
@@ -0,0 +1,561 @@
...
+
+# Swiping in .NET MAUI ListView (SfListView)
+
+## Overview
+
+The `SfListView` allows swiping items to achieve custom actions such as deleting the data, adding the data, editing the data, etc. To enable swiping, set the `SfListView.AllowSwiping` property to `true`. Swipe views are displayed when swiping from left to right or right to left (for `Horizontal` orientation, top to bottom, or bottom to top) on the item. 
+
+It provides customizable swipe templates for swiping on the left and right sides. You can restrict the layout of the swipe view up to a certain position when swiping the item by setting the `SfListView.SwipeThreshold` property. You can set the size of the swipe views by setting the `SfListView.SwipeOffset` property.
+
+N> When tapping a swiped item, the [SelectionChanging](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_SelectionChanging) and [SelectionChanged](https://help.syncfusion.com/cr/xamarin/Syncfusion.ListView.XForms.SfListView.html#Syncfusion_ListView_XForms_SfListView_SelectionChanged) events will not occur since the swiped item is reset at this time.
```
Comment (jayaleshwari) [inline MAUI/ListView/swiping.md:18]: Xamarin reference added

File: MAUI/ListView/swiping.md [hunk 10]
```diff
@@ -0,0 +1,561 @@
...
+
+  grid.Children.Add(grid1);
+  grid.Children.Add(grid2, 1, 0);
+                
+  return grid;
+});
+{%endhighlight %}
+{% endtabs %}
+
+![Customize start swipe in .NET MAUI ListView](Images/swiping/maui-listview-working-with-multiple-views-in-swipetemplate.jpg)
```
Comment (jayaleshwari) [inline MAUI/ListView/swiping.md:163]: Customize start swipe in .NET MAUI ListView -  Customize start swipe view in .NET MAUI ListView

**Review by jayaleshwari — CHANGES_REQUESTED** [review]
Please address review comments

File: MAUI/ListView/swiping.md (hunk 10 above)
Comment (RiyashameedM) [reply MAUI/ListView/swiping.md:163]: Modified

File: MAUI/ListView/swiping.md (hunk 9 above)
Comment (RiyashameedM) [reply MAUI/ListView/swiping.md:18]: Modified

**Review by jayaleshwari — APPROVED** [review]
This is fine

**Pull request merged**